src/utils/api.js

```diff
@@ -13,6 +12,11 @@ import {
     postDataSetReportCommentUrl,
     getDataSetReportFileUrls,
 } from './api/helpers'
+import {
+    getReportTablesResourceNameForSystemVersion,
+    getStandardReportsFieldsForSystemVersion,
+    formatStandardReportResponseForSystemVersion,
+} from './backwardCompatability.js'
```

Comment:
Circular dependency. I'm wondering why this hasn't caused any issues, it should have.